cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/triplestore/CgmesModelTripleStore.java

```diff
@@ -688,6 +698,8 @@ private String injectParams(String queryText, String... params) {
     private final int cimVersion;
     private final TripleStore tripleStore;
     private final QueryCatalog queryCatalog;
+    private boolean isNodeBreaker;
```

Comment:
To remove the Sonar warning (naming collision between attribute and method), we should rename attribute to nodeBreaker and nodeBreakerComputed?

Reply:
Agree, renamed the attributes. Also reduced cognitive complexity of nodeBreaker computation by splitting in two separate methods.